modules/tests/shared/src/test/scala/lucuma/core/model/SpectralDefinitionSuite.scala

```diff
@@ -5,84 +5,76 @@ package lucuma.core.model
 
 import cats.Order._
 import cats.kernel.laws.discipline._
+import coulomb._
 import eu.timepit.refined.cats._
 import lucuma.core.enum.Band
+import lucuma.core.enum.StellarLibrarySpectrum
 import lucuma.core.math.BrightnessUnits
+import lucuma.core.math.BrightnessValue
+import lucuma.core.math.Wavelength
+import lucuma.core.math.arb.ArbBrightnessValue
 import lucuma.core.math.arb.ArbRefined
+import lucuma.core.math.arb.ArbWavelength
 import lucuma.core.math.dimensional.arb.ArbMeasure
+import lucuma.core.math.units._
 import lucuma.core.model.arb._
+import lucuma.core.util.arb.ArbCollection
 import lucuma.core.util.arb.ArbEnumerated
 import monocle.law.discipline._
 import munit._
-import lucuma.core.math.Wavelength
-import coulomb._
-import lucuma.core.math.units._
+
 import scala.collection.immutable.SortedMap
-import lucuma.core.enum.StellarLibrarySpectrum
-import lucuma.core.math.BrightnessValue
 
 final class SpectralDefinitionSuite extends DisciplineSuite {
   import ArbUnnormalizedSED._
   import ArbEnumerated._
-  import ArbBandBrightness._
+  import ArbBrightnessValue._
   import BrightnessUnits._
   import ArbSpectralDefinition._
   import ArbRefined._
   import ArbMeasure._
   import ArbEmissionLine._
+  import ArbCollection._
+  import ArbWavelength._
 
   // Brightness type conversions
-  val sd1Integrated: SpectralDefinition[Integrated] = {
-    val b: BandBrightness[Integrated] =
-      BandBrightness.withDefaultUnits[Integrated](BrightnessValue.fromDouble(10.0), Band.R)
-
+  val sd1Integrated: SpectralDefinition[Integrated] =
     SpectralDefinition.BandNormalized(
       UnnormalizedSED.StellarLibrary(StellarLibrarySpectrum.A0I),
-      SortedMap(b.band -> b)
+      SortedMap(Band.R -> Band.R.brightnessMeasure[Integrated](BrightnessValue.fromDouble(10.0)))
     )
-  }
-
-  val sd1Surface: SpectralDefinition[Surface] = {
-    val b: BandBrightness[Surface] =
-      BandBrightness.withDefaultUnits[Surface](BrightnessValue.fromDouble(10.0), Band.R)
 
+  val sd1Surface: SpectralDefinition[Surface] =
     SpectralDefinition.BandNormalized(
       UnnormalizedSED.StellarLibrary(StellarLibrarySpectrum.A0I),
-      SortedMap(b.band -> b)
+      SortedMap(Band.R -> Band.R.brightnessMeasure[Surface](BrightnessValue.fromDouble(10.0)))
```

Comment:
Is the `Band.R.brightnessMeasure` just used to pick up the default units for `R` bands?  You could just as well do `Band.R` -> `Band.J.brightnessMeasure[Surface](BrightnessValue.fromDouble(10.0))` in terms of what gets created in the end?

Reply:
Yes, it's just used to pick up the default units, and yes, nothing stops us from using the default units from another band.

Maybe we should the remove the `brightnessMeasure` convenience method?

We could still create the brightness with `Band.R.defaultUnits[Surface].withValueTagged(BrightnessValue.fromDouble(10.0))`, where it is more explicit that we are just using the band's default units.

Reply:
Yes, that sounds good.  I was confused at first when I saw it, assuming that `Band.R` was somehow being associated with the measure being created there.